fhir-persistence-schema/src/test/java/com/ibm/fhir/schema/derby/DerbyMigrationTest.java

```diff
@@ -52,86 +62,121 @@ protected void setUp() throws SecurityException, IOException
 
     @Test
     public void testMigrateFhirSchema() throws Exception {
+        // The schema for each resource is the same, so we only need to
```

Comment:
@lmsurpre can you review this part? It doesn't do exactly what I think you intended and I'd like to defer to you here.